### PR TITLE
Manage provider page redesign

### PR DESCRIPTION
**Pull request by lostie**
#### What

* Re-style the the add/edit provider page for super admins/admin external users
* Add two new fields to a supplier (name and postcode)
* Resets values on the provider if the provider type is changed

#### Ticket

[Implement new 'manage provider' designs](https://dsdmoj.atlassian.net/browse/CBO-200)

### CODE REVIEW

File: app/assets/stylesheets/v2/_links.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    color: #2b8cc4;
+  }
+  @media (max-width: 640px) {
+    font-size: 16px;
+    line-height: 1.25;
+  }
+}
+
+a[target="_blank"]:after
+/*a.link-open-window:after
```
Comment (colinbruce) [inline app/assets/stylesheets/v2/_links.scss:40]: can the commented bit be removed?  herokuapp.com seems superflouous

File: config/locales/en.yml [hunk 2]
```diff
@@ -147,24 +157,17 @@ en:
         supplier_number:
           attributes:
             supplier_number:
-              invalid_format: "LGFS supplier number invalid format"
-              not_unique: "LGFS supplier number has already been taken"
+              invalid: invalid format
```
Comment (colinbruce) [inline config/locales/en.yml:160]: Is this added to a description string?

**Review by colinbruce — APPROVED** [review]
A couple of [not_a_blocker] comments but otherwise LGTM
:+1:

File: config/locales/en.yml (hunk 2 above)
Comment (lostie) [reply config/locales/en.yml:160]: Yes, the old behaviour overrode the default Rails behaviour for error messages. This one just sets the locale for the default error code when the `format` is invalid.

**Pull request merged**